### PR TITLE
remove unused settings and remove duplicates

### PR DESCRIPTION
**Pull request by vangheem**
### Description
Describe the proposed changes made in this PR.

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

**Review by bloodbare — APPROVED** [review]

File: nucliadb_node/nucliadb_node/settings.py [hunk 1]
```diff
@@ -36,13 +37,7 @@ class Settings(BaseSettings):
...
 
 
-class IndexingSettings(BaseSettings):
-    index_jetstream_target: Optional[str] = "node.{node}"
-    index_jetstream_group: Optional[str] = "node-{node}"
-    index_jetstream_stream: Optional[str] = "node"
-    index_jetstream_servers: List[str] = []
-    index_jetstream_auth: Optional[str] = None
-
+class IndexingSettings(utils_settings.IndexingSettings):
```
Comment (lferran) [inline nucliadb_node/nucliadb_node/settings.py:40]: With this change the dependency to utils is a bit bigger at the node.
If we are not planning on getting rid of it any time soon, then I think it's OK

File: nucliadb/nucliadb/ingest/settings.py [hunk 2]
```diff
@@ -39,9 +39,7 @@ def _missing_(cls, value):
                 return member
 
 
-class Settings(BaseSettings):
-    grpc_port: int = 8030
-
+class DriverSettings(BaseSettings):
     # allowing defaults is not ideal
     # TODO: implement settings without defaults
```
Comment (lferran) [inline nucliadb/nucliadb/ingest/settings.py:44]: out of curiosity: why it is not ideal?

File: nucliadb_node/nucliadb_node/settings.py (hunk 1 above)
Comment (vangheem) [reply nucliadb_node/nucliadb_node/settings.py:40]: I thought any change would be moving to rust right? So for any python changes, the utils package is our tooling package to use for this sort of thing.

We need something like that for rust though probably

File: nucliadb/nucliadb/ingest/settings.py (hunk 2 above)
Comment (vangheem) [reply nucliadb/nucliadb/ingest/settings.py:44]: IMO, things that require configuration like this should be explicit and hard fail.

For example, right now, with default redis, you also need to configure the redis to use but a user could leave this setting unfilled and wonder what is wrong when they get a bunch of redis errors.

**Review by lferran — APPROVED** [review]

**Pull request merged**